src/components/DateInput/index.jsx

```diff
@@ -181,8 +181,9 @@ const DateInput = ({
   maxDate,
   ...rest
 }) => {
-  const defaultDate =
-    typeof defaultDateInput === 'string' ? new Date(defaultDateInput) : defaultDateInput;
+  const defaultDate = moment(defaultDateInput)
```

Comment:
If no `defaultDate` is supplied you'll get '1970-01-01'.

Reply:
!! you're right!

Reply:
Or maybe it's '1969-12-31'. I have no idea anymore...